### PR TITLE
Universidade Federal de Pernambuco (Brazil)

### PR DESCRIPTION
**Pull request by damorim**
Addition of Universidade Federal de Pernambuco, Brazil.

### CODE REVIEW

File: country-info.csv [hunk 1]
```diff
@@ -150,3 +150,4 @@ York University,canada
 Zhejiang University,asia
 Bielefeld University,europe
 KTH Royal Institute of Technology,europe
+Universidade Federal de Pernambuco,southamerica
```
Comment (emeryberger) [inline country-info.csv]: For symmetry with other Brazilian institutions, please abbreviate (e.g., UFP). Thanks.

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Comment by damorim** [comment]
@emeryberger CR implemented

File: country-info.csv (hunk 1 above)
Comment (klerisson) [reply country-info.csv]: @damorim The institutional name should also be abbreviated in csrankings.csv.

**Comment by damorim** [comment]
thanks @klerisson.  just changed.

**Comment by emeryberger** [comment]
Hi - please resolve conflicts. Thanks.

**Comment by damorim** [comment]
Done.

**Comment by emeryberger** [comment]
Sorry, new conflicts to resolve!

**Comment by emeryberger** [comment]
Conflicts again :(. Please resolve and send me a separate mail and I will then merge; I will hold off on any other PRs until then.

**Comment by damorim** [comment]
Hi, I don't understand. On the link below it says no conflict (this time)
as per the message "This branch has no conflicts with the base branch". I
also fetched again from the upstream repo and did not see any change.
Perhaps I am missing something.

  https://github.com/emeryberger/CSrankings/pull/608

Marcelo

**Review by emeryberger — APPROVED** [review]

**Comment by emeryberger** [comment]
Weird - now it looks fine. <shrug>

Thanks!

**Pull request merged**